benchmark/README.md

```diff
@@ -62,7 +63,8 @@ TPCH timing results is written to stdout in the following form: `TPCH_Result,<la
 
 ## Python
 1. Upload [run_python_benchmark.sh](run_python_benchmark.sh) and all [python tpch benchmark](python/) files to the cluster.
-2. Run the benchmark by invoking:
+2. Install pyarrow and pandas on the cluster including head and worker nodes. Use Conda to install these packages.

```

Comment:
Please provide the instructions on how to do this.

Reply:
Provided more detailed instructions